test/integration/local_transactions_test.rb

```diff
@@ -51,6 +51,8 @@ class LocalTransactionsTest < ActionDispatch::IntegrationTest
         lgil_override: 8,
```

Comment:
Hmm there seems to be quite a high degree of overlap between the purpose of this integration test and the controller tests. It seems unfortunate but I'm not sure there's a quick way out of this problem - I guess this is a sign that there's either too many integration tests or that the controller tests are testing too much.

If we're not testing any scenarios in the controller tests that aren't covered by the integration tests we should just not add them.

Reply:
Yeah I wasn't 100% sure. I did find it quite useful to have all of these integration tests, but I understand they are quite cumbersome. Is it ok to keep them?

Reply:
Yeah they can be kept. There's probably no need for the country based functional tests as they're not really testing that isn't tested.

Reply:
I removed the functional country based tests (I left a NI test in there,  but it was already there before I added anything so I decided to keep it there).